src/pages/MeetingDetail.tsx

```diff
@@ -1,114 +1,41 @@
-import { css } from '@emotion/react';
+import { Suspense } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
 
-import { IconName } from '@/assets/icons';
-import { Button } from '@/components/common/Button';
-import { Card } from '@/components/common/Card';
-import { Chip } from '@/components/common/Chip';
 import { FlexBox } from '@/components/common/FlexBox';
-import { Member } from '@/components/common/Member';
 import { ScheduleLayout } from '@/components/common/ScheduleLayout';
-import { Body2, Body3, Head3 } from '@/components/common/Typography';
-import { formatDateTime } from '@/utils/formatDateTime';
-import { isPastDate } from '@/utils/isPastDate';
+import { Body3 } from '@/components/common/Typography';
+import { MeetingInfo } from '@/components/features/MeetingDetail/MeetingInfo';
+import { MemberList } from '@/components/features/MeetingDetail/MemberList';
+import { MemberScheduleCard } from '@/components/features/MeetingDetail/MemberScheduleCard';
 
 export const MeetingDetail = () => {
-  // TODO: 더미데이터 입니다. API 연결시 삭제할 예정입니다.
-  const meetingData = {
-    meetingId: 2,
-    categoryNames: ['학교', '팀플'],
-    meetingName: '세븐일레븐짱',
-    meetingStartDate: '2024-08-27',
-    meetingEndDate: '2024-08-27',
-    dueDateTime: '2024-08-26T23:59:59',
-  };
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const scheduleData: string | any[] = [
-    {
-      memberNames: ['몰라', '졸려', '배고파'],
-      startTime: '2024-08-21T17:00:00',
-      endTime: '2024-08-21T18:00:00',
-    },
-  ];
-  const memberData = {
-    anonymousStatus: false,
-    participantInfoList: [
-      {
-        nickname: '정승조',
-        votedStatus: true,
-        leaderStatus: true,
-      },
-      {
-        nickname: '정승조',
-        votedStatus: true,
-        leaderStatus: false,
-      },
-    ],
-  };
+  const { uuid } = useParams();
+  const navigate = useNavigate();
+
+  if (uuid === undefined) {
+    navigate('/');
+    return null;
+  }
```

Comment:
이러한 상황에서는 이렇게 작성할 수 있어용

```suggestion
  if (uuid === undefined) {
    return <Navigate to='/' />;
  }
```